azkaban-common/src/main/java/azkaban/executor/container/PodEventListener.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.executor.container;
+
+import static azkaban.Constants.AZ_HOST;
+import static azkaban.Constants.AZ_WEBSERVER;
+import static azkaban.Constants.EXECUTION_ID;
+import static azkaban.Constants.EXECUTOR_TYPE;
+import static azkaban.Constants.FLOW_NAME;
+import static azkaban.Constants.FLOW_STATUS;
+import static azkaban.Constants.FLOW_VERSION;
+import static azkaban.Constants.PROJECT_NAME;
+import static azkaban.Constants.SUBMIT_TIME;
+import static azkaban.Constants.SUBMIT_USER;
+import static azkaban.Constants.VERSION_SET;
+
+import azkaban.ServiceProvider;
+import azkaban.event.Event;
+import azkaban.event.EventListener;
+import azkaban.executor.ExecutableFlow;
+import azkaban.imagemgmt.version.VersionSet;
+import azkaban.spi.AzkabanEventReporter;
+import azkaban.spi.ExecutorType;
+import azkaban.utils.JSONUtils;
+import azkaban.utils.Props;
+import com.google.common.annotations.VisibleForTesting;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class is the event listener for flow executed in a Kubernetes pod
+ * It handles event, gets event meta data, and sends meta data to event reporter plugin
+ */
+public class PodEventListener implements EventListener<Event> {
+  private AzkabanEventReporter azkabanEventReporter;
+  private Props props;
+  private static final Logger logger = LoggerFactory.getLogger(PodEventListener.class);
+
+  public PodEventListener() {
+    this.props = ServiceProvider.SERVICE_PROVIDER.getInstance(Props.class);
+    try {
+      this.azkabanEventReporter = ServiceProvider.SERVICE_PROVIDER.getInstance(AzkabanEventReporter.class);
+    } catch (final Exception e) {
+      logger.info("AzkabanEventReporter is not configured");
```

Comment:
Can you please log it with logger.error?

Reply:
changed to logger.error